spec/factories/dishes.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :dish do
+    name "MyString"
+    description "MyText"
+    price 1
+    readytime "2017-01-10 16:32:43"
```

Comment:
Not a blocker but I would be mindful on how this value is set. I would prefer `Date.today (or .now or whatever..)` Also, I would consider validating that the `readytime` is in the future. (I also have a problem with the attribute itself, `readytime` does not feel like a good name. I would at least separate the words with an underscore `ready_time`)